i18n/en-GB/dashboard-income-taxation.json

```diff
@@ -4,8 +4,8 @@
         "datagovmy"
     ],
     "translation": {
-        "header": "Income Taxation",
-        "description": "Paying your taxes is a form of patriotism; a way in which every resident of Malaysia contributes to the building of our nation. This act creates a reciprocal relationship - as taxpayers, you have a right to transparency. Accordingly, this dashboard gives you a fun way to explore income tax data, with a particular focus on individual income taxes. How many people pay taxes? How much are you paying relative to other people? Does taxpaying behaviour vary across states? Dive in and find out!",
+        "header": "Income Tax",
+        "description": "Paying taxes is a responsibility to every Malaysian in contributing towards the development of the country. Accordingly, this dashboard gives you a fun way to explore income tax data, with a particular focus on individual income taxes. How many people pay taxes? How much are you paying relative to other people? Does taxpaying behaviour vary across states? Dive in and find out!",
```

Comment:
Incorrect grammar.
- "for every Malaysian" not "to"
- "to" the development, not "towards"

Why is the line on transparency removed?